src/templates/atoms.njk

```diff
@@ -651,6 +651,66 @@
             </div>
         </div>
 
+        <h4 class="u-margin-top-xl">Buttons with avatar</h4>
+        <div class="d">
+            <div>
+                <button class="a-button has-avatar">
+                    <span class="a-avatar a-avatar--primary">
+                        <span class="a-avatar__letter" aria-label="avatarTitle">A</span>
+                    </span>
+
+                    Anne-Marie
+                </button>
+
+                <button class="a-button a-button--s has-avatar">
+                    <span class="a-avatar a-avatar--primary a-avatar--s">
+                        <span class="a-avatar__letter" aria-label="avatarTitle">A</span>
+                    </span>
+
+                    Anne-Marie
+                </button>
+            </div>
+
+            <div>
+                <pre class="a-pre a-pre--scrollable"><code class="html">&lt;button class="a-button has-avatar"&gt;
+  &lt;span class="a-avatar a-avatar--primary"&gt;
+    &lt;span class="a-avatar__letter" aria-label="avatarTitle"&gt;A&lt;/span&gt;
+  &lt;/span&gt;
+
+  Anne-Marie
+&lt;/button&gt;</code></pre>
+            </div>
+        </div>
+        <div class="d">
+            <div>
+                <button class="a-button has-avatar-with-image">
+                    <span class="a-avatar">
+                        <img src="https://randomuser.me/api/portraits/women/44.jpg" alt="Avatar image." />
+                    </span>
+
+                    Anne-Marie
```

Comment:
Open vraag: Zetten we hier nog een span rond, zodat de styling kan overschreven worden?

Reply:
Zoals daarstraks besproken. Je zou dit kunnen oplossen door de CSS property die instaat voor de tekstkleur van de button te overschrijven. 😄